<?xml version="1.0" encoding="UTF-8"?>
<launch>
	
	<arg name='leader' default='leader'/> 
	<arg name='ego' default='ego'/>
	<arg name='leader_pos' default='30.0'/>
	<arg name='ego_pos' default='0.0'/>
	<arg name='laser_sensor' default='false'/>

	<include file="$(find steer)/launch/threelanes.launch"/>
	<include file="$(find sparkle)/launch/spawn.launch">
        	<arg name="robot" value="$(arg leader)"/>
		<arg name="X" value="$(arg leader_pos)"/>
        </include>
	
	<include file="$(find sparkle)/launch/spawn.launch">
        	<arg name="robot" value="$(arg ego)"/>
		<arg name="X" value="$(arg ego_pos)"/>
		<arg name="laser_sensor" value="$(arg laser_sensor)"/>
	</include>

</launch>
